composer.json

```diff
@@ -18,10 +18,10 @@
     },
     "require": {
         "php": ">=5.3",
-        "react/stream": "^1.0 || ^0.7"
+        "react/stream": "^1.2"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3 || ^5.7 || ^4.8",
-        "react/event-loop": "^1.0 || ^0.5 || ^0.4 || ^0.3"
+        "react/event-loop": "^1.2"
```

Comment:
I don't think the updates examples reference the event loop at all?

Reply:
The event loop is still necessary for some tests (see `FunctionalControlCodeParserTest.php`).